api/v1/stocks.go

```diff
@@ -16,7 +17,7 @@ func (api *APIHandler) getAllStocks(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	m := []details.StockDetails{}
+	m := make([]details.StockDetails, 20)
```

Comment:
Is there a reason for limiting this to the first 20 stocks?

Reply:
Thank you! <.<
And yes my IDE suggested me to not use empty literals for construction (though I have to be honest I'm not really sure what the downsides of doing this is, or if there even is one at all). The 20 is pretty much arbitrary and I thought since it's a Slice it can grow dynamically anyway(?).

But if this is confusing or straight up an incorrect change I will revert it back ofc! :0

Reply:
Ah I think I was confusing the make syntax for the equivelent of [20]details.StockDetails{}.
I don't really use make much but go for it :+1: I'd have guessed the compiler will optimize both equally so there shouldnt be a measurable performance difference, I might check that out!